src/patternfly/components/Button/button.scss

```diff
@@ -151,29 +152,29 @@
     bottom: 0;
     left: 0;
     content: "";
-    border: var(--pf-c-button--BorderWidth) solid;
-    border-color: var(--pf-c-button--BorderColor);
-    border-radius: var(--pf-c-button--BorderRadius);
+    border: var(--pf-c-button--after--BorderWidth) solid;
+    border-color: var(--pf-c-button--after--BorderColor);
+    border-radius: var(--pf-c-button--after--BorderRadius);
   }
 
   &:hover {
     text-decoration: none;
 
     &::after {
```

Comment:
you shouldn't need to specify this here or any of the other selectors. You can override the variable with the state selector.

Reply:
I was trying something and made that change in a PR here if it helps https://github.com/christiemolloy/patternfly-next/pull/25